sass/components/_cheader-icon.scss

```diff
@@ -0,0 +1,45 @@
+.cheader-icon {
```

Comment:
I think it's fine to just call this `channel-header-icon`. I'm in favor of self-explanatory class names even if they're a bit long. @deanwhillier thoughts?

Reply:
Sure, I can do that, we can go back to the more condensed one if we want.

Reply:
Done.